drivers/sensor/veml7700/veml7700.c

```diff
@@ -506,16 +480,6 @@ static int veml7700_init(const struct device *dev)
 		return ret;
 	}
 
-	/* Set initial data values */
-	data->thresh_low = 0;
-	data->thresh_high = 0xFFFF;
-	data->gain = VEML7700_ALS_GAIN_1_4;
-	data->it = VEML7700_ALS_IT_100;
-	data->int_mode = VEML7700_INT_DISABLED;
-	data->als_counts = 0;
-	data->als_lux = 0;
-	data->shut_down = (conf->psm != VEML7700_PSM_DISABLED) ? 0 : 1;
-
 	/* Initialize sensor configuration */
 	ret = veml7700_write_thresh_low(dev);
```

Comment:
This will fail if the device is on a power domain that isn't on at boot, and `device_is_ready` will forever return `false` on this device